server/services/prison/PrisonRegisterService.spec.ts

```diff
@@ -127,6 +127,7 @@ describe('Prison Register Service', () => {
       const prisonId = 'ASI'
 
       const expectedPrisonAddress: PrisonAddress = {
+        agencyCode: 'ASI',
```

Comment:
Some of the tests were missing the prison `agencyCode` - these are now important as the agencyCode (prison ID) is part of the CreateBarcodeRequest body object